index.html

```diff
@@ -0,0 +1,238 @@
+                    <img class="person" src="images/speaker_04.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Julia Leda</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>
+                <div class="speaker speaker-5">
+                    <img class="person" src="images/speaker_05.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Lilia Tretikov</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>
+                <div class="speaker speaker-6">
+                    <img class="person" src="images/speaker_06.png" alt="The first speaker">
+                    <div class="sp-1-details">
+                        <h3 class="sp-name">Ryan Merkley</h3>
+                        <p class="sp-details">
+                            Berkman Professor of Entrepreneural Legal Studies at Harvard Law School
+                        </p>
+                        <div class="divider"></div>
+                        <p class="footnote">
+                            Benkler Studies commoms-based peer production, and puublished his seminar book The Wealth of Networks in 2006.
+                        </p>
+                    </div>
+                </div>-->
+                <div class="more-speakers">
+                    <p class="more">More</p>
+                    <span class="entity">&#5167;</span>
+                </div>
+            </div>
+        </section>
+
+        <section class="partner-section">
+            <h3 class="partner">Partner</h3>
+            <div class="bottom-line bold"></div>
+            <ul class="partners">
+                <li class="partner-1">Microverse</li>
+                <li class="partner-1">FreeCodeCamp</li>
+                <li class="partner-1">CodeAcademy</li>
+                <li class="partner-1">Google</li>
+                <li class="partner-1">Devop</li>
+                <li class="partner-1">Odin Project</li>
+            </ul>
+        </section>
```

Comment:
- [ ] Kindly ensure the partner section is aligned the same way as in the Design. You currently have them in a vertical column. Check images below for reference
> Yours
![image](https://user-images.githubusercontent.com/90556221/169626455-e16b1211-12b4-4fec-ac0d-e2b5667040da.png)

> Design
![image](https://user-images.githubusercontent.com/90556221/169626483-8d6045ae-1ec1-4e04-9a5d-4f4dd99141b1.png)

- [ ] Additionally, try to make sure the background colors are fairly similar for this section.